src/utils/helpers.js

```diff
@@ -0,0 +1,25 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = "save",
+  loadingText = "saving..."
+) {
+  if (isLoading) {
+    //set the loading text
+    console.log(`setting text to ${loadingText}`);
+  } else {
+    //set not loading text
+  }
+}
+
+export function deleteBtnText(
+  btn,
+  isLoading,
+  defaultText = "delete",
+  loadingText = "deleting..."
+) {
+  if (isLoading) {
+    console.log(`setting text to ${loadingText}`);
+  } else {
+  }
```

Comment:
Please, implement the `setButtonText()` and `deleteBtnText()` functions correctly